tools/ClassGen/NodeGen.cpp

```diff
@@ -639,12 +639,14 @@ int main(int argc, char **argv) {
   BB.newNode("Reshape")
       .addInput("Input")
       .addMember(MemberType::VectorSizeT, "Dims")
+      .addMember(MemberType::String, "Layout")
       .addResultFromCtorArg()
       .setDocstring("Reshape the Input tensor to shape Dims.");
 
   BB.newNode("Transpose")
       .addInput("Input")
       .addMember(MemberType::VectorUnsigned, "Shuffle")
```

Comment:
It would be clunky: manipulating strings in various portions of the code: we sometimes create shuffles by doing cleaver manipulations on the mask we want. just replacing the value of certain indices.